dockers/docker-teamd/Dockerfile.j2

```diff
@@ -26,5 +26,5 @@ ENV DEBIAN_FRONTEND=noninteractive
 RUN apt-get clean -y; apt-get autoclean -y; apt-get autoremove -y
 RUN rm -rf /debs
 
-ENTRYPOINT ["/bin/bash", "-c"]
-CMD ["/usr/bin/config.sh && /usr/bin/start.sh"]
```

Comment:
Reason for that is to correctly handle signals sent to container.
See https://github.com/Azure/sonic-buildimage/issues/435